.env.dist

```
@@ -33,14 +33,22 @@ MOCK_IDP_FEDERATE_CLIENT_SECRET=federate-client-secret
+# DbInitializer Seed Data - Staging Client & User
 MOCKIDP_STAGING_CLIENT_ID=staging-mockclientid
 MOCKIDP_STAGING_CLIENT_SECRET=staging-mock-client-secret
 MOCKIDP_STAGING_CLIENT_NAME=StagingClient
 MOCKIDP_STAGING_USER_EMAIL=staging-user@example.com
 MOCKIDP_STAGING_USER_PASSWORD=password
 MOCKIDP_STAGING_REDIRECT_URI=https://ecauth-staging.azurewebsites.net/auth/callback
 
+# DbInitializer Seed Data - Production Client & User
+MOCKIDP_PRODUCTION_CLIENT_ID=production-mockclientid
+MOCKIDP_PRODUCTION_CLIENT_SECRET=production-mock-client-secret
+MOCKIDP_PRODUCTION_CLIENT_NAME=ProductionClient
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MOCKIDP_PRODUCTION_CLIENT_NAME key should go before the MOCKIDP_PRODUCTION_CLIENT_SECRET key

(UnorderedKey)